util/test_conditions

```diff
@@ -31,7 +31,7 @@ const fileSettings = {
     complianceConfig: standardComplianceParams,
   },
   ".java:": {
-    conditionRemove: /(\*\/)|(\/\*)|(@DisplayName\(\")|(\"\))/g,
+    conditionRemove: /(\*\/)|(\/\*)|(@DisplayName\(\")|(\"\))|(\")|(\"\))/g,
```

Comment:
`(\"\))` is included twice

Reply:
It is `"` and `")`
Do you have a suggestion on a better RegEx? :)
It is trying to match

`      "Precondition: Something here.")`
and turn that into
`Precondition: Something here.`

Reply:
I mean I think it should be
```suggestion
    conditionRemove: /(\*\/)|(\/\*)|(@DisplayName\(\")|(\"\))|(\")/g,
```
`(\"\))` is already in the regex once